modules/unspents/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@bitgo/unspents",
+  "version": "0.7.0",
+  "description": "Defines the chain codes used for different unspent types and methods to calculate bitcoin transaction sizes",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "build": "tsc --project tsconfig.build.json",
+    "test": "mocha --require ts-node/register \"test/**/*.ts\"",
+    "lint": "eslint --quiet .",
+    "prepare": "npm run build",
+    "unit-test": "npm run test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/BitGo/BitGoJS.git"
+  },
+  "keywords": [
+    "bitcoin",
+    "utxo"
+  ],
+  "author": "BitGo Inc.",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/BitGo/BitGoJS/issues"
+  },
+  "homepage": "https://github.com/BitGo/BitGoJS#readme",
```

Comment:
Should this point to the `unspents` specific readme? That would make sense to me since it's a module-specific package.json.